### DISCUSSION TITLE
problema com apiKey

### DISCUSSION THREAD

**Discussion by jpazul** [Biscoint/biscoint-arbitrage-bot#17] [Q&A]
quando vou dar start no cmd aparece a frase "You must specify "apiKey" in config.json" mas ela já esta especificada.

**Comment by renahnschultz** [comment]
Dentro de src tem o config.js (src/config.js) e embaixo do JSON de config tem um try que tenta fazer merge do config.json. Veja se esse código não está comentado ou foi apagado.

**Reply by andrebento302** [reply]
o meu nao funciona de jeito nenhum

2021-10-12T19:53:02.826Z [Biscoint BOT] [error] - You must specify "apiKey" in config.json
2021-10-12T19:53:02.828Z [Biscoint BOT] [error] - {}
